SqrlForNet/SqrlCommandWorker.cs

```diff
@@ -653,23 +653,25 @@ public void HelperJson()
 
             if (Options.OtherAuthenticationPaths != null && Options.OtherAuthenticationPaths.Any())
             {
-                responseMessage.Append("\"OtherUrls\":[");
-
+                responseMessage.Append(",\"OtherUrls\":[");
+                int ct = 0;
```

Comment:
ct is not a great name it would have been better have verbose wording to say what it is doing

Reply:
made the changes and pushed.